machine/reference/scp.md

```diff
@@ -27,5 +29,48 @@ A file created remotely!
 Just like how `scp` has a `-r` flag for copying files recursively,
 `docker-machine` has a `-r` flag for this feature.
 
-In the case of transferring files from machine to machine, they go through the
-local host's filesystem first (using `scp`'s `-3` flag).
\ No newline at end of file
+In the case of transferring files from machine to machine,
+they go through the local host's filesystem first (using `scp`'s `-3` flag).
+
+## Understanding file paths for remote deployments
+
+If you copy files to a remote server with `docker-machnie scp`
+in order to deploy an application (e.g., in the cloud), paths for
+files on the remote server should map to your local test environment.
+When you copy files from a local to remote system, put them in a
+path where `docker-compose` knows how to find them,
+to mount into a new container. This assumes that you will use the same model
+remotely as you do locally.
+
+For example, given this configuration for a `web` container in a
+[Compose file](/compose/compose-file/index.md), `docker-compose.yml`:
+
+```none
+web:
+  container_name: web
+  restart: always
+  build: ./web
+  expose:
+    - "8000"
+  links:
+    - postgres:postgres
+  volumes:
+    - /usr/src/app/static
+    - ./data:/usr/src/app/data
+  env_file: .env
+  command: /usr/local/bin/gunicorn --workers 4 --timeout 120 --bind :8000 app:app
+```
+
+* The first path given for [volumes](/compose/compose-file/index.md#volumes-volumedriver), `/usr/src/app/data`, is the path _inside_ the container.
+
+* The second path, `./data:/usr/src/app/data`, is the path where
```

Comment:
If you're writing this up for remote deployments, I'd shy away from the "relative path method" (i.e., `.:/foo` indicating that `.`, the current directory, should be bind-mounted into `foo`) completely. Arguably it was somewhat of a mistake to include in Compose in the first place as it's caused no end of confusion and does not accurately map to what gets sent to the Docker API. It's client side magic.

My point is that in a writeup on remote paths you should call it out explicitly that this won't work and use absolute paths. There's a big opportunity to clear up a long-term point of confusion here. I'd address it something like, "You can specify an absolute path, e.g. `/home/myuser/workspace`, which will be mounted into the container at `/workspace`, from the absolute path on the _remote host_ where the Docker daemon is running. Local client paths (e.g., your Mac laptop) will not work for daemons running on a remote VM."